src/Partners/PartnersClient.php

```diff
@@ -7,11 +7,17 @@
 use ArrowSphere\PublicApiClient\Exception\NotFoundException;
 use ArrowSphere\PublicApiClient\Exception\PublicApiClientException;
 use ArrowSphere\PublicApiClient\Partners\Entities\OrganizationUnit;
+use ArrowSphere\PublicApiClient\Partners\Entities\OrganizationUnitsResponse;
 use Generator;
 use GuzzleHttp\Exception\GuzzleException;
 
 class PartnersClient extends AbstractClient
 {
+    /**
+     * @var array
+     */
+    public $pagination = [];
```

Comment:
I don't think you need this anymore
you should not create public properties anyway